club_leader_onboarding_protocol.md

```diff
@@ -57,16 +67,17 @@ welcome to the community :-).
 
 ### During Call
 
-- [ ] Add to Slack
-  - [ ] Give them tips to introduce themselves
 - [ ] Add to GitHub Leaders team
-- [ ] Add to Facebook private group
-  - [ ] Give them tips to introduce themselves
 - [ ] Set up weekly calls
   - [ ] Talk about what the the [code of conduct][code-of-conduct] is and why
         it's important
 - [ ] Get their address and add it to their entry in the "Club Leaders" sheet in
   https://clubs.hackclub.io
+- [ ] Add to Facebook private group
+  - [ ] Give them tips to introduce themselves
+- [ ] Add to Slack
+  - [ ] Give them tips to introduce themselves
+  - [ ] Have them download the desktop app
```

Comment:
Yeah, that's a good call. Submitting a pull request for this now.

On Thu, Jan 14, 2016 at 3:52 PM, Jonathan Leung <notifications@github.com>
wrote:

> In club_leader_onboarding_protocol.md
> <https://github.com/hackclub/meta/pull/471#discussion_r49804098>:
>
> >  - [ ] Set up weekly calls
> >    - [ ] Talk about what the the [code of conduct][code-of-conduct] is and why
> >          it's important
> >  - [ ] Get their address and add it to their entry in the "Club Leaders" sheet in
> >    https://clubs.hackclub.io
> > +- [ ] Add to Facebook private group
> > +  - [ ] Give them tips to introduce themselves
> > +- [ ] Add to Slack
> > +  - [ ] Give them tips to introduce themselves
> > +  - [ ] Have them download the desktop app
>
> Should we ask them to download the the mobile app as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/meta/pull/471/files#r49804098>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta